Library/Homebrew/service.rb

```diff
@@ -474,6 +494,7 @@ def to_systemd_timer
       instance_eval(&@service_block)
       options = []
       options << "Persistent=true" if @run_type == RUN_TYPE_CRON
+      options << "OnActiveSec=0s" if @run_at_load == true && @run_type == RUN_TYPE_INTERVAL
```

Comment:
This may also be worth being a shared method.

Reply:
Does the newest commit satisfied your comment?